gordon/utils.py

```diff
@@ -183,6 +183,16 @@ def _protocol_enrich(obj):
     return settings
 
 
+def convert_cloudformation_types(data):
+    cf_data = {}
+    for k, v in six.iteritems(data):
+        if isinstance(v, Iterable):
```

Comment:
This should be `isinstance(v, Iterable) and not isinstance(v, six.string_types)` or it will join on all strings.